app/controllers/UnauthorisedController.scala

```diff
@@ -23,12 +23,12 @@ import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
 import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendController
 import views.html.unauthorised
 
-class UnauthorisedController @Inject()(
-                                        controllerComponents: MessagesControllerComponents,
-                                        unauthorisedView: unauthorised
-                                      )(implicit val appConfig: AppConfig) extends FrontendController(controllerComponents) with I18nSupport {
+class UnauthorisedController @Inject()(controllerComponents: MessagesControllerComponents,
+                                        unauthorisedView: unauthorised)
```

Comment:
@tyagikumarvineet , there is an extra space before the parameter on line 27. Can you please reformat this file?

Reply:
Reformatted